scalableminds/webknossos
# refuse tracing update actions for finished annotations

### Steps to test:
- open annotation in a tab, in another, keep the dashboard
- in dashboad, finish the annotation
- updates to the annotation should be refused (triggered reload shows read-only mode)

------
